test/python_utils/gitdist_UnitTests.py

```diff
@@ -125,17 +127,47 @@ def test_full_table(self):
       { "label" : "M", "align":"R", "fields" : ["0", "2", "25" ] },
       { "label" : "?", "align":"R", "fields" : ["0", "0", "4" ] },
       ]
-    asciiTable = createAsciiTable(tableData)
-    #print(asciiTable)
-    asciiTable_expected = \
+    table = createTable(tableData)
+    #print(table)
+    table_expected = \
       "-------------------------------------------------------------------\n" \
       "| ID | Repo Dir           | Branch | Tracking Branch | C | M  | ? |\n" \
       "|----|--------------------|--------|-----------------|---|----|---|\n" \
       "|  0 | Base: BaseRepo     | dummy  |                 |   |  0 | 0 |\n" \
       "|  1 | ExtraRepo1         | master | origin/master   | 1 |  2 | 0 |\n" \
       "|  2 | Path/To/ExtraRepo2 | HEAD   |                 |   | 25 | 4 |\n" \
       "-------------------------------------------------------------------\n"
-    self.assertEqual(asciiTable, asciiTable_expected)
+    self.assertEqual(table, table_expected)
+
+
+  def test_full_table_utf8(self):
+    if sys.version_info < (3,):
```

Comment:
@bartlettroscoe, should this `if` be inside the function or outside, such that the function doesn't exist under Python 2?

Reply:
@jmgate, I am not a Python expert but I think the if statement needs to be inside of the function.  I think what you have is fine because this function will never get called for Python versions 2.x.